src/components/Product.svelte

```diff
@@ -0,0 +1,41 @@
+<script lang="ts">
+    import {cart} from '../store/cart.js'
+    import {
+        Badge,
+        Card,
+        CardBody,
+        CardImg,
+        CardHeader,
+        CardText,
+        CardTitle,
+        Col,
+        CardFooter,
+        Button,
+    } from 'sveltestrap';
+
+    export let item
+
+</script>
+<Col class="mb-3">
+    <Card class="h-100">
+        <CardHeader>
+            <CardTitle>
+                {item.name}
+                <Badge class="bg-danger float-end">{item.price} Kč</Badge>
+            </CardTitle>
+            <CardImg src="https://drive.google.com/uc?export=view&id={item.picture}" alt="{item.name}" top="true">
+            </CardImg>
+        </CardHeader>
+        <CardBody>
+            <CardText>
+                {item.description}
+            </CardText>
+        </CardBody>
+        <CardFooter class="bg-transparent border-0">
+            <Button on:click={() => {cart.addProduct(item)}} class="float-end" color="warning">
+                Add to Cart
+            </Button>
```

Comment:
can we make use of custom events here and not use the cart store in this component?